src/go/guestagent/pkg/tracker/apitracker_test.go

```diff
@@ -128,17 +142,23 @@ func TestAddOverride(t *testing.T) {
 	// reset the exposeReq slice
 	expectedExposeReq = nil
 
+	additionalPort := "8080"
+	additionalProtocol := "udp"
```

Comment:
These should be constants, not variables, I think.

Should these be moved to the top so all the constants are defined together?

Also should `protocol` be `protocolTCP` and `additionalProtocol` be `protocolUDP`?